src/components/Book.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Book = () => (
+  <div className="wrapper">
+    <h2>Head First Javascript</h2>
+    <p>Eric Freeman</p>
+    <button type="submit">Remove Book</button>
+  </div>
+);
```

Comment:
- [ ] Please make sure the `Book` component receives the title and the author as a prop from the parent component `Books` and use it in the component instead of hardcoding them as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/201149063-4682bef7-7cc0-46ba-b7c7-dbcf3e66c136.png)